Mention Sentry wrap and code push for RN (#5483)
getsentry/sentry-docs: src/platforms/react-native/manual-setup/codepush.mdx

});
```

> If you're wrapping your root component with `Sentry.wrap` to access Performance features, `codePush` should be the outer most function because it needs access to the very root component in order to swap out the bundle.

Author: ```suggestion
If you're wrapping your root component with `Sentry.wrap` to access Performance features, `codePush` should be the outer most function because it needs access to the root component in order to swap out the bundle.
```

> In case you are wrapping your Root Component with `Sentry.wrap` to gain Performance features, `codePush` should be the outer most function because it needs access to the very root component in order to swap out the bundle.

Reviewer: I'm don't understand what you mean by "the very root component". Can you explain that or try wording it another way?

```suggestion
If you're wrapping your root component with `Sentry.wrap` to access Performance features, `codePush` should be the outer most function because it needs access to the very root component in order to swap out the bundle.
```
